src/gui/widgets/TempoSyncKnob.cpp

```diff
@@ -291,7 +291,11 @@ void TempoSyncKnob::showCustom()
 	if( m_custom == NULL )
 	{
 		m_custom = new MeterDialog( gui->mainWindow()->workspace() );
-		gui->mainWindow()->addWindowedWidget( m_custom );
+		m_subWindow = gui->mainWindow()->addWindowedWidget( m_custom );
+		Qt::WindowFlags flags = m_subWindow->windowFlags();
+		flags &= ~Qt::WindowMaximizeButtonHint;
+		m_subWindow->setWindowFlags( flags );
+        m_subWindow->setFixedSize( m_subWindow->size() );
```

Comment:
According to our coding conventions, 2 tabs should be used instead of 8 spaces.

Reply:
Fixup commit changes these spaces to tabs. Thanks!